yamscripts.yml

```diff
@@ -10,9 +10,10 @@ scripts:
 
 # bundle: Bundle assets for production
   bundle:
-    static: cpx "content-src/static/**/*" data/content -C
+    static: cpx "content-src/static/**/!(activity-streams.html)" data/content -C
```

Comment:
the reason I had to add this is that the file was getting deleted by cpx because of the `-C` argument. There is sort of a race condition and I was losing it. Any other ideas?